dbms/programs/odbc-bridge/Handlers.cpp

```diff
@@ -0,0 +1,153 @@
+#include "Handlers.h"
+#include <Common/HTMLForm.h>
+
+#include <memory>
+#include <DataStreams/IBlockOutputStream.h>
+#include <DataStreams/copyData.h>
+#include <DataTypes/DataTypeFactory.h>
+#include <Dictionaries/ODBCBlockInputStream.h>
+#include <Formats/BinaryRowInputStream.h>
+#include <Formats/FormatFactory.h>
+#include <IO/ReadBufferFromIStream.h>
+#include <IO/WriteBufferFromHTTPServerResponse.h>
+#include <IO/WriteHelpers.h>
+#include <Interpreters/Context.h>
+#include <boost/algorithm/string.hpp>
+#include <boost/tokenizer.hpp>
+#include <Poco/Ext/SessionPoolHelpers.h>
+#include <Poco/Net/HTTPServerRequest.h>
+#include <Poco/Net/HTTPServerResponse.h>
+#include <common/logger_useful.h>
+namespace DB
+{
+namespace ErrorCodes
+{
+    extern const int BAD_REQUEST_PARAMETER;
+}
+
+namespace
+{
+    std::unique_ptr<Block> parseColumns(std::string && column_string)
+    {
+        std::unique_ptr<Block> sample_block = std::make_unique<Block>();
+        auto names_and_types = NamesAndTypesList::parse(column_string);
+        for (const NameAndTypePair & column_data : names_and_types)
+            sample_block->insert({column_data.type, column_data.name});
+        return sample_block;
+    }
+
+    size_t parseMaxBlockSize(const std::string & max_block_size_str, Poco::Logger * log)
+    {
+        size_t max_block_size = DEFAULT_BLOCK_SIZE;
+        if (!max_block_size_str.empty())
+        {
+            try
+            {
+                max_block_size = std::stoul(max_block_size_str);
```

Comment:
It doesn't work as expected:

```
milovidov@milovidov-Pro-P30:~/work/ClickHouse$ g++ -std=c++17 -xc++ -include string - <<< 'int main() { return std::stoul("123abc"); }'
milovidov@milovidov-Pro-P30:~/work/ClickHouse$ ./a.out 
milovidov@milovidov-Pro-P30:~/work/ClickHouse$ echo $?
123
```

Better to use `parse` function from `ReadHelpers.h`

Reply:
Need to check for all possibly wrong usages of `std::sto...` functions in our codebase.